src/components/region-common.js

```diff
@@ -64,26 +66,38 @@ export const allRegions = [
   // In this list, us-east*, us-west*, northamerica-northeast2 and asia-northeast2 have purposefully been removed.
   // This is to avoid creating within-country silos of life sciences community data.
   // So for US, Canada and Japan, we are restricting to one region.
-  // For more information, see https://docs.google.com/document/d/1RMu8bxXAyP2q_85UHkPARW1Xpt_G_662pRi0LCML4yA/edit#heading=h.53j9ueoid5vt
-  // TODO(wnojopra): Replace the above link with the Terra support article link once published
+  // For more information, see https://support.terra.bio/hc/en-us/articles/360060777272-US-regional-versus-Multi-regional-US-buckets-trade-offs
```

Comment:
Looks like this (and `unknownRegionFlag`) no longer needs to be exported?

Reply:
Done.